docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -427,4 +427,24 @@ The value of `core_pattern` can be a literal path or it can contain a pipe symbo
   kubectl cp -n <namespace> -c yb-cleanup <yb_pod_name>:/var/yugabyte/cores/core.2334 /tmp/core.2334
   ```
 
-- If the value of `core_pattern` contains a `|` pipe symbol (for example, `|/usr/share/apport/apport -p%p -s%s -c%c -d%d -P%P -u%u -g%g -- %E`), the core dump is being redirected to a specific collector on the underlying Kubernetes node, with the location depending on the exact collector. In this case, it is your responsibility to identify the location to which these files are written and retrieve them.
\ No newline at end of file
+- If the value of `core_pattern` contains a `|` pipe symbol (for example, `|/usr/share/apport/apport -p%p -s%s -c%c -d%d -P%P -u%u -g%g -- %E`), the core dump is being redirected to a specific collector on the underlying Kubernetes node, with the location depending on the exact collector. In this case, it is your responsibility to identify the location to which these files are written and retrieve them.
+
+## Perform the follower lag check during upgrades
+
+You can use the follower lag check to ensure that the YB-Master or YB-TServer process is caught up to its peers. To find this metric on Prometheus, execute the following:
+
+```sh
+max by (instance) (follower_lag_ms{instance='<ip>:<http_port>'})
+```
+
+- *ip* represents the YB-Master IP (default: 7000) or the YB-TServer IP (default: 9000).
```

Comment:
The (default: 7000) and (default:9000) should be moved below to the HTTP port line below

Ex:  *http_port* represents the HTTP port on which the YB-Master (default: 7000) or YB-TServer (default: 9000) is listening. 

Mistake on my end for the comment on Jira